R/Data-validity.R

```diff
@@ -16,10 +16,10 @@ NULL
 #'   `ID` and `cohort` vectors which are of the same length `nObs`.
 validate_subjects <- function(object) {
   o <- Validate()
-  o$check(
-    test_int(object@nObs),
-    "nObs must be of type integer of length 1"
-  )
+  if (!test_int(object@nObs)) { # in if clause so that below test_* won't fail
```

Comment:
longer comments better in separate line above, and please use sentence style (so start with upper case letter, end with full stop). thx